pyvista/_vtk.py

```diff
@@ -171,6 +215,47 @@
         vtkTable,
         vtkUnstructuredGrid,
     )
+
+    try:
+        from vtkmodules.vtkCommonDataModel import VTK_TRIQUADRATIC_PYRAMID
+    except ImportError:  # pragma: no cover
+        pass
+
+    try:
+        from vtkmodules.vtkCommonDataModel import VTK_BEZIER_TRIANGLE
+    except ImportError:  # pragma: no cover
+        pass
+
+    try:
+        from vtkmodules.vtkCommonDataModel import VTK_BEZIER_CURVE
+    except ImportError:  # pragma: no cover
+        pass
+
+    try:
+        from vtkmodules.vtkCommonDataModel import VTK_BEZIER_QUADRILATERAL
+    except ImportError:  # pragma: no cover
+        pass
+
+    try:
+        from vtkmodules.vtkCommonDataModel import VTK_BEZIER_TETRAHEDRON
+    except ImportError:  # pragma: no cover
+        pass
+
+    try:
+        from vtkmodules.vtkCommonDataModel import VTK_BEZIER_HEXAHEDRON
+    except ImportError:  # pragma: no cover
+        pass
+
+    try:
+        from vtkmodules.vtkCommonDataModel import VTK_BEZIER_WEDGE
+    except ImportError:  # pragma: no cover
+        pass
+
+    try:
+        from vtkmodules.vtkCommonDataModel import VTK_BEZIER_PYRAMID
+    except ImportError:  # pragma: no cover
+        pass
```

Comment:
I assume at least the BEZIER ones are either all present or all absent. Should we merge these into a single `try` block?

Reply:
Good idea! Thanks :)
```suggestion
    try:
        from vtkmodules.vtkCommonDataModel import VTK_TRIQUADRATIC_PYRAMID
        from vtkmodules.vtkCommonDataModel import VTK_BEZIER_TRIANGLE
        from vtkmodules.vtkCommonDataModel import VTK_BEZIER_CURVE
        from vtkmodules.vtkCommonDataModel import VTK_BEZIER_QUADRILATERAL
        from vtkmodules.vtkCommonDataModel import VTK_BEZIER_TETRAHEDRON
        from vtkmodules.vtkCommonDataModel import VTK_BEZIER_HEXAHEDRON
        from vtkmodules.vtkCommonDataModel import VTK_BEZIER_WEDGE
        from vtkmodules.vtkCommonDataModel import VTK_BEZIER_PYRAMID
    except ImportError:  # pragma: no cover
        pass
```